tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -188,3 +230,45 @@ var _ = ginkgov2.Describe("VM backup and restore tests", ginkgov2.Ordered, func(
 		}, nil),
 	)
 })
+
+// Temporary VM-specific copy of runRestore. This is here to avoid making big
+// changes to runRestore when they are likely to be taken out in the near future.
+func runVmRestore(brCase VmBackupRestoreCase, backupName, restoreName string, nsRequiresResticDCWorkaround bool) {
+	log.Printf("Creating restore %s for case %s", restoreName, brCase.Name)
+	restore, err := lib.CreateRestoreFromBackup(dpaCR.Client, namespace, backupName, restoreName)
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	gomega.Eventually(lib.IsRestoreDone(dpaCR.Client, namespace, restoreName), timeoutMultiplier*time.Minute*60, time.Second*10).Should(gomega.BeTrue())
+	// TODO only log on fail?
+	describeRestore := lib.DescribeRestore(veleroClientForSuiteRun, dpaCR.Client, restore)
+	ginkgov2.GinkgoWriter.Println(describeRestore)
+
+	restoreLogs := lib.RestoreLogs(kubernetesClientForSuiteRun, dpaCR.Client, restore)
+	restoreErrorLogs := lib.RestoreErrorLogs(kubernetesClientForSuiteRun, dpaCR.Client, restore)
+	accumulatedTestLogs = append(accumulatedTestLogs, describeRestore, restoreLogs)
+
+	if !brCase.SkipVerifyLogs {
+		gomega.Expect(restoreErrorLogs).Should(gomega.Equal([]string{}))
+	}
+
+	// Check if restore succeeded
+	succeeded, err := lib.IsRestoreCompletedSuccessfully(kubernetesClientForSuiteRun, dpaCR.Client, namespace, restoreName)
+	if brCase.RestoreErr != nil {
+		gomega.Expect(succeeded).To(gomega.Equal(false))
+		gomega.Expect(err).NotTo(gomega.BeNil())
+		gomega.Expect(err.Error()).To(gomega.ContainSubstring(brCase.RestoreErr.Error()))
+	} else {
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		gomega.Expect(succeeded).To(gomega.Equal(true))
+	}
+
+	if nsRequiresResticDCWorkaround {
```

Comment:
Is this still needed?

Reply:
I think possibly not, but I'm not really sure. I just copied all of this from the original runRestore, since it is planned to be temporary.